api/build.gradle

```diff
@@ -12,3 +13,11 @@ dependencies {
 javadoc {
     exclude 'io/opentelemetry/internal/**'
 }
+
+jar {
+    inputs.property("moduleName", moduleName)
+
+    manifest {
+        attributes  'Automatic-Module-Name': moduleName
+    }
+}
```

Comment:
Missing empty newline at the end of these.

Reply:
Please fix this everywhere.

Reply:
I've added an empty new line to the root build.gradle, as it did not have one (prior to my involvement). All other files no longer have been touched at the end of the file, so they remain as they were.